capstone/src/main/java/com/google/sps/servlets/StudentServlet.java

```diff
@@ -26,4 +26,29 @@ private static String convertToJsonUsingGson(Student student) {
     String json = gson.toJson(student);
     return json;
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get student object based on the logged in email
+    UserService userService = UserServiceFactory.getUserService();
+    String userEmail = userService.getCurrentUser().getEmail();
+    Student student = PrototypeStudents.PROTOTYPE_STUDENTS.get(userEmail);
+
+    if (request.getParameter("action").equals("join")) {
```

Comment:
nit: let's define these literals as constants since they'll be referred to often